pkg/tests/ossm/bug_istiopods_test.go

```diff
@@ -51,6 +55,51 @@ func TestIstiodPodFailsAfterRestarts(t *testing.T) {
 	})
 }
 
+func TestIstiodPodFailsWithValidationMessages(t *testing.T) {
+	NewTest(t).Groups(Full, Disconnected, ARM).Run(func(t TestHelper) {
+		t.Log("Verify that Istio pod is not failing when validationMessages was enabled")
+
+		oc.RecreateNamespace(t, meshNamespace)
+		oc.ReplaceOrApplyString(t, meshNamespace, template.Run(t, validationMessagesSMCP, DefaultSMCP()))
+		t.Cleanup(func() {
+			oc.RecreateNamespace(t, meshNamespace)
+		})
+
+		istiodPod := pod.MatchingSelector("app=istiod", meshNamespace)
+		oc.WaitPodRunning(t, istiodPod)
+		retry.UntilSuccessWithOptions(t, retry.Options().MaxAttempts(10), func(t TestHelper) {
+			oc.LogsFromPods(t, meshNamespace, "app=istiod", assert.OutputContains(
+				"successfully acquired lease istio-system/istio-analyze-leader",
+				"Successfully acquired lease for analyzer in istiod pod",
+				"Expected to acquire lease for analyzer in istiod pod, but was not",
+			),
+			)
+		})
+
+		t.Log("Verify that Istiod pod doesn't contain SIGSEGV when validationMessages was enabled")
+		t.Log("Reference: https://issues.redhat.com/browse/OSSM-6177")
+
+		oc.LogsFromPods(t, meshNamespace, "app=istiod", assert.OutputDoesNotContain(
+			"SIGSEGV: segmentation violation",
+			"Found no SIGSEGV",
+			"Expected to find no SIGSEGV, but got some SIGSEGV",
+		),
+		)
```

Comment:
There is no delay between the time when the controller acquires the lease and the assertion here. It might make sense to wait a few seconds before checking.

Reply:
You are right. 5 seconds would be enough, wouldn't be?